Adds Dutch Locale

gedejong opened the pull request:
- Thanks to Jacob van Lingen to provide the initial translation for
login messages
- Not a full translation yet.

Commit:
KEYCLOAK-5307 Dutch locale

- Thanks to Jacob van Lingen to provide the initial translation for
login messages

hmlnarik commented:
Thank you. Please update the PR by rebasing to the current master to fix the failing test.

gedejong commented:
@hmlnarik I've rebased the commit and removed the dutch admin messages. Since > 90% of the dutch population and every administrator is able to understand and speak English, adding a translation of the admin console will only introduce confusion. Many of he English terms are specific to the domain and would not benefit from translation, leaving 'stray' Dutch terms intermixed with English jargon.

So, the benefits are small relatively large impact of translating all messages and keeping it up-to-date. I see the German locale has done something similar.

hmlnarik commented:
@gedejong Thank you, I think that approach is allright.

Unfortunately, we are not able to maintain this translation ourselves. Would you be able to maintain it and update it when it's needed?

gedejong commented:
I understand the need for maintainers. Within reasonable time-frames, I'm willing to maintain the translation. It would be nice, however, if there is another dutch committer to Keycloak who is willing to check my translations.

hmlnarik commented:
Thank you.

gedejong commented:
(btw, I just fixed a typo in my master branch, amended the commit and did a force push, since it is not merged in the main repo anyways)

sebastienblanc commented:
I'm half dutch and the translation "ziet er goed uit" ;) 
:+1: to merge

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+oauthGrantRequest=Wilt u deze toegangsrechten verlenen?
+inResource=in
+
+emailVerifyInstruction1=Een e-mail met instructies om uw e-mailadres te verifiÃ«ren is zojuist verzonden.
```
Not sure if this is a rendering artifact by github or UTF8 gone wrong. Above there is an "ë" in "Creëer realm" that is rendered correctly though, so one of the two is probably broken.

gedejong replied:
Agreed, part of the translation was done by Jacob van Lingen. Probably something went wrong in the transport.

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+copyCodeInstruction=Kopieer deze code en plak deze in uw applicatie:
+
+personalInfo=Persoonlijke informatie:
+role_realm-admin=Realm Beheerder
```
Previously you used "realmbeheer". Either way the space should go, or you should go for "Realm beheren" in both cases.

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+role_manage-clients=Beheer clients
+role_manage-events=Beheer gebeurtenissen
+role_view-profile=Profiel bekijken
+role_read-token=Lees token
```
"Token lezen"

gedejong replied:
:+1:

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+invalidEmailMessage=Ongeldig e-mailadres.
+accountDisabledMessage=Account is uitgeschakeld, neem contact op met beheer.
+accountTemporarilyDisabledMessage=Account is tijdelijk uitgeschakeld, neem contact op met beheer of probeer het later opnieuw.
+expiredCodeMessage=Login time-out. Gelieve opnieuw in te loggen.
```
"Login verlopen" / "Maximale loginduur verstreken" / "Logintijd verlopen"?

gedejong replied:
:+1:

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+missingEmailMessage=Voer uw e-mailadres in.
+missingUsernameMessage=Voer uw gebruikersnaam in.
+missingPasswordMessage=Voer uw wachtwoord in.
+missingTotpMessage=Voer uw authenticator code in.
```
"authenticator code" -> "authenticatiecode" (also below)

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+missingTotpMessage=Voer uw authenticator code in.
+notMatchPasswordMessage=Wachtwoorden komen niet overeen.
+invalidPasswordExistingMessage=Ongeldig bestaand wachtwoord.
+invalidPasswordConfirmMessage=Wachtwoordbevestiging komt niet overeen.
```
Wachtwoord komt niet overeen met wachtwoordbevestiging

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+usernameExistsMessage=Gebruikersnaam bestaat al.
+emailExistsMessage=E-mail bestaat al.
+
+federatedIdentityExistsMessage=Gebruiker met {0} {1} bestaat al. Log in met het beheer-account om het account te koppelen.
```
beheer-account -> beheerdersaccount?

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+verifyEmailMessage=U moet uw e-mailadres verifiëren om uw account te activeren.
+linkIdpMessage=U moet uw e-mailadres verifiëren om uw account te koppelen aan {0}.
+
+emailSentMessage=U zult binnenkort een e-mail ontvangen met verdere instructies.
```
"U zult binnenkort ... ontvangen" -> "U ontvangt binnenkort"

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+failedLogout=Afmelden is mislukt
+unknownLoginRequesterMessage=De login requester is onbekend
+loginRequesterNotEnabledMessage=De login requester is niet geactiveerd
+bearerOnlyMessage=Bearer-only applicaties mogen geen browser login initiëren
```
"browser login" -> "browserlogin" (like on the line below)

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+bearerOnlyMessage=Bearer-only applicaties mogen geen browser login initiëren
+standardFlowDisabledMessage=Client is mag geen browserlogin starten met het opgegeven response_type. Standard flow is uitgeschakeld voor de client.
+implicitFlowDisabledMessage=Client is mag geen browserlogin starten met opgegeven response_type. Implicit flow is uitgeschakeld voor de klant.
+invalidRedirectUriMessage=Ongeldige omleiding uri
```
"Ongeldige redirect-URI"

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+unsupportedNameIdFormatMessage=Niet-ondersteunde NameIDFormat
+invalidRequesterMessage=Ongeldige requester
+registrationNotAllowedMessage=Registratie is niet toegestaan
+resetCredentialNotAllowedMessage=Het resetten van de aanmeldgegevens is niet toegestaan
```
"resetten" -> "opnieuw instellen"?

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+invalidRequesterMessage=Ongeldige requester
+registrationNotAllowedMessage=Registratie is niet toegestaan
+resetCredentialNotAllowedMessage=Het resetten van de aanmeldgegevens is niet toegestaan
+permissionNotApprovedMessage=Toestemming niet goedgekeurd.
```
Not sure what this means. This message also seems to be unused in the code, so no matter.

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+permissionNotApprovedMessage=Toestemming niet goedgekeurd.
+noRelayStateInResponseMessage=Geen relay state in antwoord van de identity provider.
+insufficientPermissionMessage=Onvoldoende rechten om identiteiten te koppelen.
+couldNotProceedWithAuthenticationRequestMessage=Het authenticatie verzoek naar de identity provider wordt afgebroken.
```
"authenticatie verzoek" -> "authenticatieverzoek"

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+noRelayStateInResponseMessage=Geen relay state in antwoord van de identity provider.
+insufficientPermissionMessage=Onvoldoende rechten om identiteiten te koppelen.
+couldNotProceedWithAuthenticationRequestMessage=Het authenticatie verzoek naar de identity provider wordt afgebroken.
+couldNotObtainTokenMessage=Kan het token van identity provider niet ophalen.
```
Kon geen token bemachtigen van de identity provider

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+couldNotProceedWithAuthenticationRequestMessage=Het authenticatie verzoek naar de identity provider wordt afgebroken.
+couldNotObtainTokenMessage=Kan het token van identity provider niet ophalen.
+unexpectedErrorRetrievingTokenMessage=Onverwachte fout bij het ophalen van de token van de identity provider.
+unexpectedErrorHandlingResponseMessage=Onverwachte fout bij het verwerken van de response van de identity provider.
```
"response" -> "respons"

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+unexpectedErrorRetrievingTokenMessage=Onverwachte fout bij het ophalen van de token van de identity provider.
+unexpectedErrorHandlingResponseMessage=Onverwachte fout bij het verwerken van de response van de identity provider.
+identityProviderAuthenticationFailedMessage=Verificatie mislukt. Er kon niet worden geauthenticeerd met de identity provider.
+identityProviderDifferentUserMessage=U bent geauthenticeerd als {0}, maar u werd als {1} verwacht geauthenticeerd te zijn
```
"maar u werd als {1} verwacht" -> "maar u werd verwacht als {1}"

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+unexpectedErrorHandlingResponseMessage=Onverwachte fout bij het verwerken van de response van de identity provider.
+identityProviderAuthenticationFailedMessage=Verificatie mislukt. Er kon niet worden geauthenticeerd met de identity provider.
+identityProviderDifferentUserMessage=U bent geauthenticeerd als {0}, maar u werd als {1} verwacht geauthenticeerd te zijn
+couldNotSendAuthenticationRequestMessage=Kan het authenticatie verzoek niet verzenden naar de identity provider.
```
"authenticatie verzoek" -> "authenticatieverzoek" (also below)

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+unexpectedErrorHandlingRequestMessage=Onverwachte fout bij het verwerken van het authenticatie verzoek naar de identity provider.
+invalidAccessCodeMessage=Ongeldige toegangscode.
+sessionNotActiveMessage=Sessie niet actief.
+invalidCodeMessage=Er is een fout opgetreden, log daarom opnieuw in.
```
"log daarom opnieuw in" -> "probeer nogmaals in te loggen"

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+sessionNotActiveMessage=Sessie niet actief.
+invalidCodeMessage=Er is een fout opgetreden, log daarom opnieuw in.
+identityProviderUnexpectedErrorMessage=Onverwachte fout tijdens de authenticatie met de identity provider
+identityProviderNotFoundMessage=Kan geen identity provider vinden die aan de identifier voldoet.
```
Geen identity provider gevonden met deze naam

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+identityProviderUnexpectedErrorMessage=Onverwachte fout tijdens de authenticatie met de identity provider
+identityProviderNotFoundMessage=Kan geen identity provider vinden die aan de identifier voldoet.
+identityProviderLinkSuccess=Uw account is met succes gekoppeld aan {0} account {1}.
+staleCodeMessage=Deze pagina is niet langer geldig. Ga terug naar uw applicatie en log opnieuw in, alstublieft
```
"Ga terug ..." -> "Keer terug naar uw applicatie om opnieuw in te loggen"

scranen commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+realmSupportsNoCredentialsMessage=Realm ondersteunt geen enkel soort aanmeldgegeven.
+identityProviderNotUniqueMessage=Realm ondersteunt meerdere identity providers. Er kon niet bepaald worden welke identity provider er gebruikt zou moeten worden tijdens de authenticatie.
+emailVerifiedMessage=Uw e-mailadres is geverifieerd.
+staleEmailVerificationLink=De link die u gebruikt is verlopen en daarom niet langer geldig. Misschien heeft u uw uw e-mailadres al geverifieerd?
```
"De link die u gebruikt is niet langer geldig, wellicht omdat u uw e-mailadres al eerder geverifieerd heeft."

scranen reviewed (commented):
Would be intelligible in its current form, but I added some notes for the author's consideration. In abbreviated form:

Contractions of Dutch and English words should not be separated by spaces (see https://onzetaal.nl/taaladvies/engelse-woorden-in-nederlandse-samenstellingen).

Imperative form (gebiedende wijs) and regular encouragements in present tense (aansporingen) are mixed, sometimes in the same message. I recommend going for imperative form. See also https://onzetaal.nl/taaladvies/red-redt-de-tijger.

The phrasing in the different templates of identical messages is not always consistent.

More in general, I noticed that some words seem randomly capitalised ("Mobile Authenticator", "Keycloak Accountbeheer", "Nieuw Wachtwoord").

hmlnarik commented:
@scranen @sebastienblanc Thanks for review
@gedejong Would you be reflecting on @scranen's comments?

gedejong commented on themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties:
```diff
@@ -0,0 +1,133 @@
+doAdd=Voeg toe
+doSignOut=Afmelden
+editAccountHtmlTitle=Bewerk account
+federatedIdentitiesHtmlTitle=Verenigde identiteiten
```
agreed

gedejong commented on themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties:
```diff
@@ -0,0 +1,133 @@
+street=Straat
+locality=Stad of plaats
+region=Staat, provincie of regio
+postal_code=Zip- of postcode
```
I wanted to stay close to the English translation, which includes both forms (Zip or Postal code). Nevertheless, the other translations seems to go for "Postcode". I'll change it as well.

gedejong commented on themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties:
```diff
@@ -0,0 +1,133 @@
+emailVerified=Emailadres geverifieerd
+gssDelegationCredential=GSS gedelegeerde aanmeldgegevens
+role_admin=Beheer
+role_realm-admin=Realm Beheer
```
I'm going with Realmbeheer.

gedejong commented on themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties:
```diff
@@ -0,0 +1,133 @@
+role_manage-account-links=Beheer accountkoppelingen
+role_read-token=Lees token
+role_offline-access=Offline toegang
+role_uma_authorization=Verkrijg toestemmingen
```
that's what I thought too, but the English message is: "Obtain permissions". Italian translation, for example, is "Ottieni permess". "Permissie" is a Dutch word (no Anglicization) so I'll change "Toestemming" to "Permissie".

gedejong commented on themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties:
```diff
@@ -0,0 +1,133 @@
+fullAccess=Volledige toegang
+offlineToken=Offline Token
+revoke=Toekenning terugtrekken
+configureAuthenticators=Ingestelde authenticators
```
The other translations go both ways. The English message is: ```configureAuthenticators=Configured Authenticators``` which I'm basing this translation off.

gedejong commented on themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties:
```diff
@@ -0,0 +1,133 @@
+totpStep1=Installeer <a href="https://freeotp.github.io/" target="_blank">FreeOTP</a> of Google Authenticator op uw apparaat. Beide toepassingen zijn beschikbaar in <a href="https://play.google.com">Google Play</a> en de Apple App Store.
+totpStep2=Open de toepassing en scan de QR-code of voer de sleutel in.
+totpStep3=Voer de door de toepassing gegeven eenmalige code in en klik op Opslaan om de configuratie af te ronden.
+missingUsernameMessage=Geef gebruikersnaam alstublieft.
```
I agree with you, but at the same time, I'm trying to stay close to the English translation: "Please specify username". Well, I'll factor it to your suggestion, which seems more in line with what I see online.

gedejong commented on themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties:
```diff
@@ -0,0 +1,133 @@
+invalidPasswordConfirmMessage=Wachtwoordbevestiging komt niet overeen.
+invalidTotpMessage=Ongeldige authenticatiecode.
+emailExistsMessage=Emailadres bestaat reeds.
+readOnlyUserMessage=U kunt uw account niet updaten aangezien het account alleen-lezen is.
```
done

gedejong commented on themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties:
```diff
@@ -0,0 +1,133 @@
+invalidTotpMessage=Ongeldige authenticatiecode.
+emailExistsMessage=Emailadres bestaat reeds.
+readOnlyUserMessage=U kunt uw account niet updaten aangezien het account alleen-lezen is.
+readOnlyPasswordMessage=U kunt uw wachtwoord niet veranderen omdat uw account alleen-lezen is.
```
Wel, wachtwoord wijzigen wilde weer eens niet, want uw account is read-only :)

gedejong commented on themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties:
```diff
@@ -0,0 +1,133 @@
+readOnlyPasswordMessage=U kunt uw wachtwoord niet veranderen omdat uw account alleen-lezen is.
+successTotpMessage=Mobiele authenticator geconfigureerd.
+successTotpRemovedMessage=Mobiele authenticator verwijderd.
+successGrantRevokedMessage=Gunning succesvol ingetrokken
```
Agreed

gedejong commented on themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties:
```diff
@@ -0,0 +1,133 @@
+invalidFederatedIdentityActionMessage=Ongeldige of ontbrekende actie op verenigde identiteit.
+identityProviderNotFoundMessage=Gespecificeerde identity provider niet gevonden.
+federatedIdentityLinkNotActiveMessage=Deze verenigde identiteit is niet langer geldig.
+federatedIdentityRemovingLastProviderMessage=U kunt de laatste verenigde identiteit niet verwijderen aangezien u het wachtwoord niet heeft.
```
I understand the meaning now, but that's not in the English translation: `You can''t remove last federated identity as you don''t have password.`

gedejong replied:
Ik heb het aangepast naar:`U kunt de laatste federated identity provider niet verwijderen aangezien u dan niet langer kunt inloggen.`

gedejong commented on themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties:
```diff
@@ -0,0 +1,133 @@
+invalidPasswordMinUpperCaseCharsMessage=Ongeldig wachtwoord: het moet minstens {0} hoofdletters bevatten.
+invalidPasswordMinSpecialCharsMessage=Ongeldig wachtwoord: het moet minstens {0} speciale karakters bevatten.
+invalidPasswordNotUsernameMessage=Ongeldig wachtwoord: het mag niet met de gebruikersnaam overeenkomen.
+invalidPasswordRegexPatternMessage=Ongeldig wachtwoord: het voldoet niet aan de reguliere expressie(s).
```
Agreed, I tried to stay close to the English message.

gedejong commented on themes/src/main/resources-community/theme/base/email/messages/messages_nl.properties:
```diff
@@ -0,0 +1,27 @@
+eventLoginErrorSubject=Inlogfout
+eventLoginErrorBody=Er is een mislukte inlogpoging gedetecteerd op uw account om {0} vanuit {1}. Als u dit niet was, neemt u dan contact op met de beheerder.
+eventLoginErrorBodyHtml=<p>Er is een mislukte inlogpoging gedetecteerd op uw account om {0} vanuit {1}. Als u dit niet was, neemt u dan contact op met de beheerder.</p>
+eventRemoveTotpSubject=TOTP verwijderd
```
I agree, but again I tried to keep close to the other translations (both English, but also all other languages). I suggest we don't change it here, but have a change request (JIRA issue) to modify all translations.

gedejong commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+doDecline=Afwijzen
+doForgotPassword=Wachtwoord vergeten?
+doClickHere=Klik hier
+doImpersonate=Belichamen
```
Completely agreed. I considered that translation and many others, but somehow ended up with the creepiest one.

gedejong commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+impersonateTitleHtml=<strong>{0}</strong> Belichaam gebruiker</strong>
+realmChoice=Realm
+unknownUser=Onbekende gebruiker
+loginTotpTitle=Mobile Authenticator Setup
```
yes, but then I'd have to translate Authenticator as well (at least, that's how it feels to me). "Mobile Authenticator" or "Mobiele Authenticatie-app" (or sth like that). That's why I kept it close to the original and close to what I think most users might know.

gedejong commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+realmChoice=Realm
+unknownUser=Onbekende gebruiker
+loginTotpTitle=Mobile Authenticator Setup
+loginProfileTitle=Update account informatie
```
:+1:

gedejong commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+unknownUser=Onbekende gebruiker
+loginTotpTitle=Mobile Authenticator Setup
+loginProfileTitle=Update account informatie
+loginTimeout=U bent te lang bezig geweest met inloggen. Het loginproces begint overnieuw.
```
:+1:

gedejong commented on themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:
```diff
@@ -0,0 +1,224 @@
+oauthGrantTitleHtml={0}
+errorTitle=Er is een fout opgetreden...
+errorTitleHtml=Er is een fout opgetreden...
+emailVerifyTitle=Emailadres verificatie
```
:+1:

gedejong commented on themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties:
```diff
@@ -0,0 +1,133 @@
+successTotpRemovedMessage=Mobiele authenticator verwijderd.
+successGrantRevokedMessage=Gunning succesvol ingetrokken
+accountUpdatedMessage=Uw account is gewijzigd.
+accountPasswordUpdatedMessage=Uw wachtwoord is veranderd.
```
I changed all veranderd to gewijzigd

gedejong commented:
Ok, CR comments are all processed. Thank you so much @scranen and @sebastienblanc for the CR. Hopefully you can go through the last changes one more time.

scranen commented on themes/src/main/resources-community/theme/base/email/messages/messages_nl.properties:
```diff
@@ -1,27 +1,27 @@
 identityProviderLinkSubject=Koppel {0}
-identityProviderLinkBody=Iemand wil uw "{1}" account koppelen met "{0}" account van gebruiker {2}. Indien u dit was, klik dan op de onderstaande link om de accounts te koppelen\n\n{3}\n\nDeze link zal over {4} minuten vervallen.\n\nIndien u de accounts niet wilt koppelen, negeert u dan dit bericht. Als u accounts koppelt, dan kunt in naar {1} inloggen via {0}.
-identityProviderLinkBodyHtml=<p>Iemand wil uw "{1}" account koppelen met "{0}" account van gebruiker {2}. Indien u dit was, klik dan op de onderstaande link om de accounts te koppelen</p><p><a href="{3}">Link om accounts te koppelen</a></p><p>Deze link zal over {4} minuten vervallen.</p><p>Indien u de accounts niet wilt koppelen, negeert u dan dit bericht. Als u accounts koppelt, dan kunt in naar ...
+identityProviderLinkBody=Iemand wil uw "{1}" account koppelen met "{0}" account van gebruiker {2}. Als u dit was, klik dan op de onderstaande link om de accounts te koppelen\n\n{3}\n\nDeze link zal over {4} minuten vervallen.\n\nAls u de accounts niet wilt koppelen, negeer dan dit bericht. Als u accounts koppelt, dan kunt u bij {1} inloggen via {0}.
```
Quotes around "{1}" are not present in other texts (see below).

gedejong replied:
true, but better to change this consistently with the other translations as well.

scranen commented on themes/src/main/resources-community/theme/base/email/messages/messages_nl.properties:
```diff
@@ -1,27 +1,27 @@
 passwordResetSubject=Wijzig wachtwoord
-passwordResetBody=Iemand heeft zojuist gevraagd om de aanmeldgegevens van {2} account te wijzigen. Indien u dit was, klik dan op de onderstaande koppeling om ze te wijzigen.\n\n{0}\n\nDe link en de code zullen binnen {1} minuten vervallen.\n\nIndien u uw aanmeldgegevens niet wilt wijzigen, negeert u dan dit bericht en er zal niets veranderd worden.
-passwordResetBodyHtml=<p>Iemand heeft zojuist gevraagd om de aanmeldgegevens van {2} account te wijzigen. Indien u dit was, klik dan op de onderstaande koppeling om ze te wijzigen.</p><p><a href="{0}">Wijzig aanmeldgegevens</a></p><p>De link en de code zullen binnen {1} minuten vervallen.</p><p>Indien u uw aanmeldgegevens niet wilt wijzigen, negeert u dan dit bericht en er zal niets veranderd wor ...
+passwordResetBody=Iemand verzocht de aanmeldgegevens van {2} account te wijzigen. Als u dit was, klik dan op de onderstaande koppeling om ze te wijzigen.\n\n{0}\n\nDe link en de code zullen binnen {1} minuten vervallen.\n\nAls u uw aanmeldgegevens niet wilt wijzigen, negeer dan dit bericht en er zal niets gewijzigd worden.
```
"van {2} account" --> "van uw {2} account" (also below).

gedejong replied:
Small change "van" => "van uw". Quotes, well, I agree, but consistency seems more important here. Also merged changes from master.

scranen reviewed (approved):
2 minor comments, but nothing I care deeply about.

gedejong commented:
@scranen almost there, I fixed 1 of the minor comments, the other one is... well, it depends on consistency. I agree with the comment, but it would not be consistent with the English translation, nor any of the other translations.

hmlnarik commented:
@gedejong Hi, thank you for the work. To merge the PR, could you please rebase to current master and squeeze the commits into single ones with a commit message starting with "KEYCLOAK-5307 "? I believe then it would be possible to merge it.

gedejong commented:
@hmlnarik Hi, I rebased the commits to the latest master and squashed them together into one reworded commit.
